apps/web/src/app/(private)/_lib/createSdk.ts

```diff
@@ -23,16 +21,18 @@ async function getLatitudeApiKey() {
 }
 
 export async function createSdk({
+  workspace,
   projectId,
   apiKey,
   __internal,
 }: {
+  workspace: Workspace
   projectId?: number
   apiKey?: string
   __internal?: { source: LogSources }
-} = {}) {
+}) {
   if (!apiKey) {
-    const result = await getLatitudeApiKey()
+    const result = await getLatitudeApiKey(workspace)
```

Comment:
No need to fetch again workspace when is already fetched by the users of this function. Passing it by argument